### DISCUSSION TITLE
Advice please.  Copy, prune and re-organize old imported repository

### DISCUSSION THREAD

**Discussion by BryTack** [community/community#101559] [New to GitHub] [answered]
Very grateful for any suggestions on how to go about this. Assume I know nothing about GIT .. you will not be to far wrong.

I've imported from a very old Software Version Control system a 'legacy' repository.  This contains information about file changes going back 20 years and products that are now meaningless.  While I want to keep this, I want to create a new 'meaningful' repository from it. Containing only fraction of the legacy one. 

 How do I create a new hierarchical structure, 'cut and copy' between repositories, delete unwanted and re-organise to get a new repositories?  Normally use GitKraken but doesn't seem to do this. Better tool??  Any 'online' information out there?

Many thanks
Westby

**Comment by Shahnawazk7944** [comment, marked as answer]
Hi @BryTack 
GitKraken is excellent for managing current Git repositories, but it was not intended for the type of restructuring you require. Here are two tools you can explore.
Github Desktop's Importer (Simple and free, good for smaller migrations) and SmartGit (Comprehensive migration tool with advanced filtering and restructuring features and SmarGit documentation have detailed instructions on using SmarGit for repository migration)

**Reply by BryTack** [reply]
Thank you.  Look likes should check these tools out.

**Reply by 7blacktiger7** [reply]
> Hi @BryTack 
> GitKraken is excellent for managing current Git repositories, but it was not intended for the type of restructuring you require. Here are two tools you can explore.
> Github Desktop's Importer (Simple and free, good for smaller migrations) and SmartGit (Comprehensive migration tool with advanced filtering and restructuring features and SmarGit documentation have detailed instructions on using SmarGit for repository migration)

**Reply by 7blacktiger7** [reply]
Good work

**Comment by Sarahsma523** [comment]
I would like to get some support on how to have all my entities with a one n to one person and a single person that can help me get my money back
